### PR TITLE
deprecate package key for service parameters, use full type instead

### PR DESCRIPTION
**Pull request by dirk-thomas**
Replaces #249.

CI packaging build: [Build Status](https://ci.ros2.org/job/ci_packaging_linux/354/)

### CODE REVIEW

**Comment by dirk-thomas** [comment]
@mikaelarguedas Can you please give this patch a try. Thanks.

**Comment by mikaelarguedas** [comment]
I won't be behind a computer for the long weekend so I'll be able to test this patch on Monday at the earliest.

File: src/parameter_bridge.cpp [hunk 1]
```diff
@@ -108,15 +112,33 @@ int main(int argc, char * argv[])
...
+        }
+      }
       printf(
-        "Trying to create bridge for ROS 2 service '%s' "
-        "with package '%s' and type '%s'\n",
-        service_name.c_str(), package_name.c_str(), type_name.c_str());
+        "Trying to create bridge for ROS 2 service '%s' with type '%s'\n",
+        service_name.c_str(), type_name.c_str());
 
+      size_t index = type_name.find("/");
```
Comment (hidmic) [inline src/parameter_bridge.cpp:132]: @dirk-thomas nit^N: can be made `const`

File: src/parameter_bridge.cpp [hunk 2]
```diff
@@ -154,15 +174,34 @@ int main(int argc, char * argv[])
...
+        }
+      }
       printf(
-        "Trying to create bridge for ROS 1 service '%s' "
-        "with package '%s' and type '%s'\n",
-        service_name.c_str(), package_name.c_str(), type_name.c_str());
+        "Trying to create bridge for ROS 1 service '%s' with type '%s'\n",
+        service_name.c_str(), type_name.c_str());
+
+      size_t index = type_name.find("/");
```
Comment (hidmic) [inline src/parameter_bridge.cpp:194]: @dirk-thomas nit^N: can be made `const`

**Comment by mikaelarguedas** [comment]
Tested his on top of the nightly image and it works fine in both directions :+1:

Parameter file used for testing:
```yaml
---
# service requests going from 2 to 1
# server on ROS1 side, client on ROS2 side
services_2_to_1:
  -
    service: /add_two_ints
    # package: roscpp_tutorials
    type: roscpp_tutorials/TwoInts
# service requests going from 1 to 2
# server on ROS2 side, client on ROS1 side
services_1_to_2:
  -
    service: /add_two_ints
    type: example_interfaces/srv/AddTwoInts

```

File: src/parameter_bridge.cpp (hunk 1 above)
Comment (dirk-thomas) [reply src/parameter_bridge.cpp:132]: Done in 5ea5a40c23a65cef995caba7d8334ed482d65c8f.

File: src/parameter_bridge.cpp (hunk 2 above)
Comment (dirk-thomas) [reply src/parameter_bridge.cpp:194]: Done in 5ea5a40c23a65cef995caba7d8334ed482d65c8f.

**Pull request merged**